src/test/java/edu/wpi/first/outlineviewer/view/dialog/AddBooleanDialogTest.java

```diff
@@ -11,6 +12,7 @@
   }
 
   @Test
+  @Tag("NonWindowsTest")
```

Comment:
Sure, what happens though? And can you reproduce this on a windows OS that is non appveyor?

Reply:
This happens on all windows machines I have tried. The mouse clicks just to the left of the slider and does not hit it.